learning/scripting/gdscript/gdscript_styleguide.rst

```diff
@@ -0,0 +1,179 @@
+.. _doc_gdscript_styleguide:
+
+GDScript Style Guide
+====================
+
+Description
+-----------
+
+This styleguide lists conventions to write elegant GDScript. The goal is
+to encourage writing clean, readable code and promote consistency across
+projects, discussions, and tutorials. Hopefully, this will also
+encourage development of auto-formatting tools.
+
+Since GDScript is close to Python, this guide is inspired by Python's
+`PEP 8 <https://www.python.org/dev/peps/pep-0008/>`__ programming
+styleguide.
+
+.. note:: Godot's built-in script editor uses a lot of these conventions
+          by default. Let it help you.
+
+Code Structure
+--------------
+
+Indentation
+~~~~~~~~~~~
+
+Indent type: Tabs *(editor default)*
+
+Indent size: 4 *(editor default)*
+
+Each indent level should be one greater than the block containing it.
+
+**Good**
+
+::
+
+    for i in range(10):
+        print("hello")
+
+**Bad**
+
+::
```

Comment:
Missing linebreak so the code block doesn't render

Reply:
Fixed.